app/views/pages/create-an-account.html.slim

```diff
@@ -24,12 +25,18 @@ p
   ' If you don't have a <a href="https://github.com" target="_blank">GitHub</a> account yet, please create one.
 
 img
-  = image_tag(("pages/help/sign-in-link.png"), class: "help-image")
+  = image_tag(("pages/help/nav-bar-sign-in.png"), class: "help-image")
 
 h2 2. Fill out your profile
 
 p
-  ' Once signed in, you should update your profile information. There are a few mandatory fields:
+  ' Once signed in, you should update your profile information.
```

Comment:
can you change this part to the following and remove the image?

“once signed in for the first time, you will be redirected to the ‘Edit your profile’ section. In order to keep using the app, you MUST fill out your profile with the mandatory fields (see list below). Don't forget to click on the ‘Save’ button to save your profile — otherwise you will not be able to proceed.”

The problem seems to be that users log in and disregard the “edit profile” screen they get. Then they just click elsewhere in the app, and are automatically logged out/unable to do anything.